rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/JobInfoData.java

```diff
@@ -93,6 +93,10 @@
 
     private List<String> preciousTasks;
 
+    private Long parentId = null;
```

Comment:
The default value of the parentId is set to -1 in the SchedulerRestInterface.java (line 230), and here it is null.
It would be better to have a homogenize the default value of parentId.

Reply:
I would like to keep it null in the database, I will look at the SchedulerRestInterface default

Reply:
@nebilbenmabrouk in SchedulerRestInterface the default value of parentId is in revisionAndJobInfo, to filter only jobs which are children of the given parentId. It is not the same semantic as JobInfoData